vignettes/getting-started.Rmd

```diff
@@ -25,6 +25,9 @@ library(gimap)
 example_data <- example_data()
 
 # Let's examine this example metadata
+
+# TODO: ## this may or may not be the right place to do it, but perhaps a more descriptive comment of what we're looking at would help. e.g. "Let's examine this example pgPEN counts table, with column IDs representing time points at which gDNA was extracted and the experimental (biological/technical?) replicate number" 
```

Comment:
@danieljgroso this is a great idea! Do you mind writing out a brief description of the data here?

Reply:
Leaving this here for my reference; RStudio is being quite buggy and not sure if it saved

# Let's examine this example pgPEN counts table. It's divided into columns containing: 
# - an ID corresponding to the names of paired guides
# - gRNA sequence 1, targeting "paralog A"
# - gRNA sequence 2, targeting "paralog B"
# - The sample, day, and replicate number for which gRNAs were sequenced

Reply:
@cansavvy I went ahead and pushed that description!